pkg/lifecycle/render/github/render.go

```diff
@@ -279,7 +281,7 @@ func getDestPathNoProxy(asset api.GitHubAsset, builder *templates.Builder) (stri
 		}
 	}
 
-	return filepath.Join(constants.InstallerPrefixPath, destDir, assetPath), nil
+	return filepath.Join(renderRoot, destDir, assetPath), nil
```

Comment:
This is the change that fixes it, the rest is passing this down through the call stack. Aware that this value is very much tramp data at the moment, but don't have a solution to that off the top of my head.